### ISSUE TITLE
Week 1 - Refactor accessibility challenge to provide clarity around tabindex

### ISSUE THREAD

**Issue by Samatar26** [foundersandcoders/master-reference#292]
[The accessibility challenge](https://github.com/foundersandcoders/accessibility-challenge) should have more of a focus on accessibility rather than tab-index. In FAC10's SGC, there was a lot of confusion about tab-index and the example was unclear as it was were input fields which are tab-able b y default. The mentor also just wanted to point to the existence of tab-index and not delve into the topic.

**Comment by jsms90** [comment]
It would be great if you could introduce tabindex in a better way though, since this is something that is not widely well-implemented. Also worth looking more into aria roles

Some helpful resources:
+ [WebAIM on tabindex](http://webaim.org/techniques/keyboard/tabindex)
+ [guide from google](https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex)
+ [accessible culture on tabindex & aria roles](http://accessibleculture.org/articles/2010/05/tabindex/) (I haven't actually read through this yet)
+ [egghead tutorial](https://egghead.io/lessons/html-5-using-the-tabindex-attribute-for-keyboard-accessibility)

**Comment by jsms90** [comment]
@Samatar26 I've just re-watched the egghead tutorial (above), that should give you some excellent ideas for what you could add to @njsfield's morning challenge

**Comment by jsms90** [comment]
@Samatar26 @antoniotrkdz How is this going?

**Comment by Samatar26** [comment]
@jsms90 Finishing up the solutions branch, Nick is on standby to merge as soon as I'm done. The Egghead video is best thing ever 👍

**Comment by jsms90** [comment]
Isn't it just! :heart_eyes:

**Comment by jsms90** [comment]
@Samatar26 Ready to close?

**Comment by Samatar26** [comment]
@jsms90 Yes!

**Comment by jsms90** [comment]
FAC11 still had lots of confusion over tabindex and now people were adding it to `p` elements.

I've submitted a couple of PRs to the workshop, to remove the current references to `tabindex` but we still need to find a way of putting `tabindex` into week 1 in a way that shows people what it's useful for e.g. modals
